rules/linux/defense_evasion_hidden_shared_object.toml

```diff
@@ -24,12 +24,12 @@ If enabling an EQL rule on a non-elastic-agent index (such as beats) for version
 risk_score = 47
 rule_id = "766d3f91-3f12-448c-b65f-20123e9e9e8c"
 severity = "medium"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "Defense Evasion"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Defense Evasion", "Elastic Endgame"]
 timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-file where event.action : "creation" and file.extension == "so" and file.name : ".*.so"
+file where event.type : "creation" and file.extension == "so" and file.name : ".*.so"
```

Comment:
Was `event.action` originally `creation`? In your picture it appears like its `file_create_event`.

Reply:
We are changing this one to `event.type` because of that difference, `event.action` is different between the platform